core-bundle/src/Resources/contao/dca/tl_article.php

```diff
@@ -564,7 +564,7 @@ public function addIcon($row, $label)
 			Image::getPath(rtrim($image, '_') . '_')
 		);
 
-		$href = System::getContainer()->get('router')->generate('contao_backend_preview', array('page'=>$row['pid'], 'article'=>($row['alias'] ?: $row['id'])));
+		$href = System::getContainer()->get('router')->generate('contao_backend_preview', array('page'=>$row['pid'], 'article'=>($row['alias'] ?? $row['id'])));
```

Comment:
But that does change the behavior if `$row['alias']` is set to the empty string, in this case the URL generation would fail I presume.

Reply:
Reverted in 0b1812a8230b8be33dd305dc5e5ac3d7412e0272.